get-started/get-started-resources/get-started-resources.adoc

```diff
@@ -0,0 +1,47 @@
+= Get Started Resources
```

Comment:
Something is awkward about this section title, maybe change to "Getting Started Resources" or "Resources for Getting Started" or just "Resources"

Reply:
Good idea. I'll change that.